packages/opentelemetry-core/src/platform/BaseAbstractPlugin.ts

```diff
@@ -36,6 +38,7 @@ export abstract class BaseAbstractPlugin<T> implements Plugin<T> {
   protected _logger!: Logger;
   protected _moduleExports!: T;
   protected _tracer!: Tracer;
+  protected _meter!: Meter;
```

Comment:
just for `_meter`, Since the `MeterProvider` is optional in the `enable` method, maybe we can initialize it from the global meter provider. Then in the `enable` method, we can get a new meter if the meter provider was actually passed.

Reply:
This seems like a good compromise